Added gems

LaurieLinz opened the pull request:
Broke the large commit into several smaller ones.

[#117117283]

jasonnoble commented:
Oh my!  Can you amend this commit and follow the 50/72 character limit per line? :)

Source: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

jasonnoble commented on Guardfile:
```diff
@@ -0,0 +1,37 @@
+  files.each { |file| watch(helper.real_path(file)) }
+end
+
+guard :rspec, all_on_start: true, all_after_pass: true,
```
🎉

LaurieLinz replied:
You are saying to take out: cmd: 'bundle exec rspec' do
?

Am I understanding correctly?

Laurie

> On Apr 7, 2016, at 1:59 PM, Jason Noble notifications@github.com wrote:
> 
> In Guardfile https://github.com/LaurieLinz/hyperbaric_medicine/pull/1#discussion_r58934808:
> 
> > +#
> > +# and, you'll have to watch "config/Guardfile" instead of "Guardfile"
> > +
> > +guard :bundler do
> > -  require 'guard/bundler'
> > -  require 'guard/bundler/verify'
> > -  helper = Guard::Bundler::Verify.new
> >   +
> > -  files = ['Gemfile']
> > -  files += Dir['*.gemspec'] if files.any? { |f| helper.uses_gemspec?(f) }
> >   +
> > -  # Assume files are symlinked from somewhere
> > -  files.each { |file| watch(helper.real_path(file)) }
> >   +end
> >   +
> >   +guard :rspec, all_on_start: true, all_after_pass: true,
> >   🎉
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub https://github.com/LaurieLinz/hyperbaric_medicine/pull/1/files/be5fba3da83e1881ac1a866a34da0b2a4f106216#r58934808

jasonnoble replied:
No, I was celebrating your changes. :)

LaurieLinz replied:
Sorry I am confused about doing some thing right!  JK.  Thanks for all the help.  

> On Apr 7, 2016, at 3:07 PM, Jason Noble notifications@github.com wrote:
> 
> In Guardfile https://github.com/LaurieLinz/hyperbaric_medicine/pull/1#discussion_r58945297:
> 
> > +#
> > +# and, you'll have to watch "config/Guardfile" instead of "Guardfile"
> > +
> > +guard :bundler do
> > -  require 'guard/bundler'
> > -  require 'guard/bundler/verify'
> > -  helper = Guard::Bundler::Verify.new
> >   +
> > -  files = ['Gemfile']
> > -  files += Dir['*.gemspec'] if files.any? { |f| helper.uses_gemspec?(f) }
> >   +
> > -  # Assume files are symlinked from somewhere
> > -  files.each { |file| watch(helper.real_path(file)) }
> >   +end
> >   +
> >   +guard :rspec, all_on_start: true, all_after_pass: true,
> >   No, I was celebrating your changes. :)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub https://github.com/LaurieLinz/hyperbaric_medicine/pull/1/files/be5fba3da83e1881ac1a866a34da0b2a4f106216#r58945297

jasonnoble commented:
1,357 lines of changes is a lot for one commit.  You could break these out into separate commits (one for Rspec/Guard, one for bootstrap, one for quiet_assets and backtrace filter, etc).

LaurieLinz committed:
Added Gems:

rspec-rails
guard-bundler
guard-rspec
guard-migrate
factory_girl_rails
capybara
shoulda-matchers
quiet_assets

[#117117283]

LaurieLinz committed:
Initialized Guard and Rspec.
Added —fail-fast and —format documentation

[#117117283]

LaurieLinz committed:
Configured Shoulda-Matchers and Capybara.
 Added backtrace to spec helper.

[#117117283]

LaurieLinz committed:
Added and configured Bootstrap.

[#117117283]

jasonnoble commented:
Looks good to me...

LaurieLinz committed:
Added launchy and faker gems.
Uncommented bcrypt in Gemfile.

[#117117283]
